source/manual/alerts/enhanced-ecommerce.html.md

```diff
@@ -0,0 +1,23 @@
+---
+owner_slack: "#search-team"
+title: Enhanced ecommerce data export
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2017-06-27
+review_in: 1 months
+---
+
+This process is related to collection of data for analytics and is run daily so that data can be uploaded to GA for newly created content. This process can be rerun multiple times without side effect.
```

Comment:
Just occurred to me: does running it several times fill up the disk? When does the cleanup job run?